studio/src/pages/[organizationSlug]/[namespace]/graph/[slug]/playground.tsx

```diff
@@ -556,6 +565,7 @@ const PlaygroundPage: NextPageWithLayout = () => {
                 showAttribution: false,
               }),
             ]}
+            schema={isLoading ? null : parseSchema(data?.sdl) ?? undefined}
```

Comment:
Will "parseSchema" need to run on every render?
Would it be better to use something like useMemo to parse it only once?

Reply:
Yep, moved it inside a useMemo